Dockerfile

```diff
@@ -67,17 +69,25 @@ RUN apt-get update -q && apt-get install -q -y --no-install-recommends \
 
 # Compile asn1scc
 RUN git clone https://github.com/ttsiodras/asn1scc.git \
-    && cd asn1scc && dotnet build "asn1scc.sln"
+    && cd asn1scc \
+    && git checkout 4.2.5.1f \
+    && dotnet build "asn1scc.sln"
 
 # Install opengeode
 RUN git clone https://gitrepos.estec.esa.int/taste/opengeode.git \
     && cd opengeode \
     && PATH=~/.local/bin:"${PATH}" && pyside6-rcc opengeode.qrc -o opengeode/icons.py &&  python3 -m pip install --upgrade .
 
+# Download RTEMS and compile example C application
+RUN wget -q https://rtems-qual.io.esa.int/public_release/rtems-6-sparc-gr712rc-smp-4.tar.xz \
+    && tar -xf rtems-6-sparc-gr712rc-smp-4.tar.xz -C /opt \
+    && rm -f rtems-6-sparc-gr712rc-smp-4.tar.xz \
+    && cd /opt/rtems-6-sparc-gr712rc-smp-4/src/example \
+    && make
```

Comment:
I think it would be nice if we could add RTEMS bin folder to PATH

Reply:
ack